package.json

```diff
@@ -93,39 +95,19 @@
     "jest-environment-jsdom": "^25.0.0",
     "nock": "^13.0.7",
     "prettier": "^2.2.1",
+    "rimraf": "^3.0.2",
     "sinon": "^9.2.4",
     "ts-jest": "^26.5.2",
     "typedoc": "^0.20.32",
     "typescript": "~4.2.2"
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "ts",
-      "json",
-      "node"
-    ],
-    "transform": {
-      "^.+\\.tsx?$": "ts-jest"
-    },
-    "testMatch": [
-      "**/*.test.ts",
-      "**/*.test.js"
-    ],
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "tests/"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 100,
-        "functions": 100,
-        "lines": 100,
-        "statements": 100
-      }
-    },
-    "setupFiles": [
-      "./tests/setupTests.ts"
-    ]
+  "lavamoat": {
+    "allowScripts": {
+      "@lavamoat/preinstall-always-fail": false,
+      "core-js": true,
```

Comment:
Nit: Isn't this post-install script just a console message? :thinking: Or is it something more. We may as well set this to `false` if that's all that it is, to reduce the surface area for attacks.